apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -1735,7 +1796,14 @@ export class CaseNotificationService extends BaseNotificationService {
       linkEnd: '</a>',
     })
 
-    return this.sendEmail(subject, html, name, email, undefined, !link)
+    return this.sendEmail({
+      subject,
+      html,
+      recipientName: name,
+      recipientEmail: email,
+      attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.